src/components/LooMap/LooMap.js

```diff
@@ -23,11 +27,25 @@ const LooMap = ({
   controlsOffset,
   showCrosshair,
 }) => {
+  const theme = useTheme();
   const mapRef = React.useRef();
 
+  useEffect(() => {
+    const map = mapRef.current.leafletElement.getContainer();
+
+    // when focused on the map container, Leaflet allows the user to pan the map by using the arrow keys
+    // without the application role screen reader software overrides these controls
+    // this also avoid the entire main region being announced
+    map.setAttribute('role', 'application');
+    map.setAttribute('aria-label', 'use arrow keys to pan the map');
+    map.setAttribute(
+      'aria-keyshortcuts',
+      'ArrowUp ArrowDown ArrowLeft ArrowRight'
+    );
+  });
+
```

Comment:
I think we will only need to apply these on the first render (unless they are overidden?) in which case a empty dependecy array should be passed as the second argument to useEffect